### PR TITLE
Updated weekly wiki

### PR DESCRIPTION
**Pull request by nfs0619**

### CODE REVIEW

File: docs/Seeed_Elderly/weekly_wiki/wiki240819.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
+---
+description: Weekly Wiki
+title: Weekly Wiki
+keywords:
+- weeely wiki
```
Comment (gmacario) [inline docs/Seeed_Elderly/weekly_wiki/wiki240819.md]: Maybe is it "weekly" instead?

File: docs/Seeed_Elderly/weekly_wiki/wiki240819.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+This example connects the `Usr` pin to `D3`, and then uses XIAO ESP32S3 to detect whether the button is pressed.
+
+
+
+
+
+
+
+## Existed Wiki Updated
```
Comment (gmacario) [inline docs/Seeed_Elderly/weekly_wiki/wiki240819.md]: Maybe "Existing"?

File: docs/Seeed_Elderly/weekly_wiki/wiki240819.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+
+### [reComputer R1000 act as OPC UA server with Node-RED](https://wiki.seeedstudio.com/recomputer_r1000_nodered_opcua_server/)
+
+The OPC Unified Architecture (UA), introduced in 2008, represents a significant evolution in industrial communication standards. Unlike its predecessors under the OPC Classic umbrella, OPC UA is a platform-independent, service-oriented architecture that unifies all the functionalities of previous OPC specifications into a single, extensible framework. This modern approach not only preserves the core capabilities of OPC Classic but also enhances them with a multi-layered design that meets the complex demands of today’s industrial environments.
+
+### [Micropython with XIAO ESP32](https://wiki.seeedstudio.com/xiao_esp32c3_with_micropython/)
```
Comment (gmacario) [inline docs/Seeed_Elderly/weekly_wiki/wiki240819.md]: MicroPython (upper case P)

File: docs/Seeed_Elderly/weekly_wiki/wiki240819.md [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+
+<div align="center"><img width={800} src="https://files.seeedstudio.com/wiki/esp32c3_micropython/title.png" /></div>
+
+[MicroPython](https://github.com/micropython/micropython/wiki) is a Python interprer with a partial native code compilation feature. It provides a subset of Python 3.5 features, implemented for embedded processors and constrained systems. It is different from CPython and you can read more about the differences [here](https://github.com/micropython/micropython/wiki/Differences).
+
+- [Micropython with XIAO ESP32C3](https://wiki.seeedstudio.com/xiao_esp32c3_with_micropython/)
+- [Micropython with XIAO ESP32S3](https://wiki.seeedstudio.com/xiao_esp32s3_with_micropython/)
+
+### [Circuitpython with XIAO ESP32](https://wiki.seeedstudio.com/xiao_esp32c3_with_circuitpython/)
```
Comment (gmacario) [inline docs/Seeed_Elderly/weekly_wiki/wiki240819.md]: CircuitPython (upper case P)

**Review by MatthewJeffson — APPROVED** [review]
Thanks for the update! and the replacement!

**Pull request merged**